pages/c/_cluster/_product/projectsnamespaces.vue

```diff
@@ -0,0 +1,227 @@
+
+      return uniq(ids);
+    },
+    projectsWithoutNamespaces() {
+      return this.projects.filter(project => !this.projectIdsWithNamespaces.includes(project.name));
+    },
+    // We're using this because we need to show projects as groups even if the project doesn't have any namespaces.
+    rowsWithFakeNamespaces() {
+      const fakeRows = this.projectsWithoutNamespaces.map((project) => {
+        return {
+          groupByLabel: `${ ('resourceTable.groupLabel.notInAProject') }-${ project }`,
+          isFake:          true,
+          mainRowKey:      project.id,
+          project,
+        };
+      });
+
+      return [...this.rows, ...fakeRows];
+    },
+    createProjectLocation() {
+      return {
+        name:   'c-cluster-product-resource-create',
+        params: {
+          product:  this.$store.getters['currentProduct'].name,
+          resource: MANAGEMENT.PROJECT
+        },
+      };
+    },
+    groupPreference: mapPref(GROUP_RESOURCES),
+    filteredRows() {
+      return this.groupPreference === 'none' ? this.rows : this.rowsWithFakeNamespaces;
+    }
+  },
+  methods: {
+    slotName(project) {
+      return `main-row:${ project.id }`;
+    },
+    createNamespaceLocation(group) {
+      const project = group.rows[0].project;
+
+      return {
+        name:   'c-cluster-product-resource-create',
+        params: {
+          product:  this.$store.getters['currentProduct'].name,
+          resource: NAMESPACE
+        },
+        query: { [PROJECT_ID]: project?.metadata.name }
+      };
+    },
+    showProjectAction(event, group) {
+      const project = group.rows[0].project;
+
+      this.$store.commit(`action-menu/show`, {
+        resources: [project],
+        elem:      event.target
+      });
+    },
+    showProjectActionButton(group) {
+      const project = group.rows[0].project;
+
+      return !!project;
+    },
+    projectLabel(group) {
+      const row = group.rows[0];
+
+      if (row.isFake) {
+        return this.t('resourceTable.groupLabel.project', { name: row.project?.nameDisplay }, true);
+      }
+
+      return row.groupByLabel;
+    },
+
+    projectDescription(group) {
+      const project = group.rows[0].project;
+
+      return project?.description;
+    }
+  }
+};
+</script>
+
+<template>
+  <Loading v-if="$fetchState.pending" />
+  <div v-else class="project-namespaces">
+    <Masthead
+      :schema="projectSchema"
+      :type-display="t('projectNamespaces.label')"
+      :resource="MANAGEMENT.PROJECT"
+      :create-location="createProjectLocation"
+      :create-button-label="t('projectNamespaces.createProject')"
+    />
+    <ResourceTable
+      class="
+      table"
+      v-bind="$attrs"
+      :schema="schema"
+      :headers="headers"
+      :rows="filteredRows"
+      :groupable="true"
+      group-tooltip="resourceTable.groupBy.project"
+      key-field="_key"
+      v-on="$listeners"
+    >
+      <template #group-by="group">
+        <div class="project-bar">

```

Comment:
So I believe this may be related to the tabs not showing up in dark mode. Something Vince just pointed out. I don't think that this should be centered but I think there was some room for improvement to align the tab text with the button text.

Are you okay with this

![image](https://user-images.githubusercontent.com/55104481/118017056-fe27d400-b30a-11eb-8741-cd00fc38971f.png)

![image](https://user-images.githubusercontent.com/55104481/118017131-1697ee80-b30b-11eb-9f32-19a634ab0a9c.png)

Reply:
Good point, in light mode this looks ok. Let's leave it as is, i think @lvuch was looking into the dark mode tab and I'll check this out again after those changes are made